tests/python/common/test_heterograph-apply-edges.py

```diff
@@ -260,6 +278,23 @@ def _print_error(a, b):
             _test(lhs, rhs, binary_op)
 
 
+# Here we test heterograph with only 1 source node because the format of node
```

Comment:
this comment looks not clear to me. `only single source node type`? clearly describe the data structure as well?

Reply:
OK, I have modified it.